core/src/main/resources/static/media/img/en_GB/app-store-badge.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is in the folder ``en_GB``, but our default locale is en-US, so that seems wrong.
What about extending the unit test with an assert for this URL?